css-align-3/resources/alignment-parsing-utils.js

```diff
@@ -0,0 +1,19 @@
+var position = [ "start", "end", "left", "right", "center", "flex-start", "flex-end"];
```

Comment:
Oops, sorry I see you added them now.

PS: I'm still getting used to all this GitHub and review process. :smile: